exercises/strings/rot13/rot13.js

```diff
@@ -29,14 +29,35 @@
  */
 
 function rot13(string) {
-  // This is your job. :)
+  let finalString = ''
+  
+  for (character of string) {  
+    let newAscii = character.charCodeAt(string)
+    let newCharCode = character.charCodeAt(string)
+    
+    if (newAscii >= 65 && newAscii < 78 || newAscii >= 97 && newAscii < 110) {
+      newCharCode = Number(newAscii) + 13
+      finalString = finalString + String.fromCharCode(newCharCode)
```

Comment:
You can do`finalString += String.fromCharCode(newCharCode)` to simplify your code! (Same on lines 45 and 50)